readme.md

```diff
@@ -63,6 +63,7 @@ Elements in beautiful READMEs include, but are not limited to: images, screensho
 
 - [Common Readme](https://github.com/noffle/common-readme#readme) - A common readme style for Node. Includes a guide and a readme generator.
 - [Feedmereadmes](https://github.com/lappleapple/feedmereadmes#readme) - README editing and project analysis/feedback.
+- [Glow - Read README in Terminal](https://github.com/charmbracelet/glow#readme) - Display Markup in Terminal. Usefull for Developer Hints.
```

Comment:
*Useful

Reply:
Also, this description is written in title case, when it should be in sentence case in order to be consistent with the rest of the list.